libraries/adaptive-expressions/src/builtinFunctions/formatDateTime.ts

```diff
@@ -31,16 +32,19 @@ export class FormatDateTime extends ExpressionEvaluator {
      * @private
      */
     private static evaluator(): EvaluateExpressionDelegate {
-        return FunctionUtils.applyWithError((args: any[]): any => {
+        return FunctionUtils.applyWithOptionsAndError((args: any[], options: Options): any => {
```

Comment:
`unknown` instead of `any`, if possible.

Reply:
fixed. below same issues are fixed in the same way.